com.microsoft.mrtk.core/Interactables/StatefulInteractable.cs

```diff
@@ -149,66 +126,46 @@ public string SpeechRecognitionKeyword
                 }
             }
         }
-
-        [SerializeField]
-        [Tooltip("If true, then the voice command will only respond to voice commands while this Interactable has focus.")]
-        private bool voiceRequiresFocus = true;
-
+        
         /// <summary>
         /// Does the voice command require this to have focus?
         /// If true, then the voice command will only respond to voice commands while this Interactable has focus.
         /// </summary>
-        public bool VoiceRequiresFocus
-        {
-            get => voiceRequiresFocus;
-        }
+        [field: SerializeField, FormerlySerializedAs("voiceRequiresFocus"),
+            Tooltip("If true, then the voice command will only respond to voice commands while this Interactable has focus.")]
+        public bool VoiceRequiresFocus { get; private set; } = true;
 
         #endregion Settings
 
         #region Public state
 
-        [Obsolete("Please use Selectedness() instead.")]
-        public float PinchAmount => Selectedness();
-
-        [SerializeField]
-        [EditableTimedFlag]
-        [Tooltip("Is the interactable toggled?")]
-        private TimedFlag isToggled = new TimedFlag();
-
         /// <summary>
         /// Is the interactable toggled?
         /// </summary>
-        public TimedFlag IsToggled => isToggled;
+        [field: SerializeField, EditableTimedFlag, FormerlySerializedAs("isToggled"), Tooltip("Is the interactable toggled?")]
```

Comment:
Mostly a syntax question for me: do _each_ of these need to be prepended with `field:`, or is it good enough to just put it once at the beginning when it's within one `[ ]` scope?

Reply:
I.... think just one `field:` is fine, because the yaml does indeed serialize with the `<PropName>k__BackingField` name